account/applications/methods/postApplications.md

```diff
@@ -14,18 +14,23 @@ Bandwidth's Account API leverages Basic Authentication with your Dashboard API C
+| `CallStatusMethod`                    | No                         | HTTP method for events sent to the `CallStatusCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>                                                        |
+| `CallbackCreds`                       | No, but highly recommended | Basic auth credentials to apply to your message & voice events                                                                                                                                                                     |
+| `CallbackCreds.UserId`                | No, but highly recommended | Basic auth `UserId`                                                                                                                                                                                                                |
+| `CallbackCreds.Password`              | No, but highly recommended | Basic auth `Password`                                                                                                                                                                                                              |
+| `CallInitiatedFallbackUrl`            | No, for `Voice-V2`         | Url to receive [voice events](../../../voice/bxml/callbacks/about.md) URL is used when messages fail to arrive at `CallInitiatedCallbackUrl`                                                                                       |
+| `callInitiatedFallbackMethod`         | No, for `Voice-V2`         | HTTP method for events sent to the `CallInitiatedFallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>                                                     |
+| `CallInitiatedFallbackCreds`          | No, for `Voice-V2`         | Basic auth credentials to apply to voice fallback events                                                                                                                                                                           |
+| `CallInitiatedFallbackCreds.UserId`   | No, for `Voice-V2`         | Basic auth `UserId` for fallback credentials                                                                                                                                                                                       |
+| `CallInitiatedFallbackCreds.Password` | No, for `Voice-V2`         | Basic auth `Password` for fallback credentials                                                                                                                                                                                     |
```

Comment:
So these are required for messaging, but not for Voice-V2? That doesn't seem correct

Reply:
I'm not sure how these have been marked as required for Messaging, I had meant to mark them as not required, but for use only with Voice-V2 apps.

Reply:
I'd just mark them "No" then, as they are not mandatory for either Voice or Messaging apps

Reply:
Is there question about whether they're mandatory/optional/prohibited/ignored for messaging?

Reply:
The consequences/requirements for using the Fallback fields in a Voice app are different than in a Messaging app. Don't we need to convey that it's only applicable to Voice apps?

Reply:
Per conversation with Lucas, we'll just mark mandatory as "No" for all the new fields. Their descriptions provide the details that it's for voice. This is the same way `CallStatusCallbackUrl` handles it today.